Week3/project3/project.js

```diff
@@ -0,0 +1,60 @@
+const url = "https://api.github.com/orgs/HackYourFuture/repos?per_page=100";
+  const HyfReposHttps = async ()=>{
+    const response = await axios(url);
+    return response;
+  }
+  async function check(){
+   const data= await HyfReposHttps();
+   if(data.status==200){
+    SelectElement(data)
+   }else{
+     console.log("there is an error")
+   }
+  }
+  check()
+async function SelectElement(data){
+  console.log(data)
+    let selectElement = document.getElementById('repositories');
```

Comment:
I think it's better to select the element outside the function just make it as a global variable, so you can reuse the variable in other functions in `line 27` for example